test/controllers/graphql_controller_test.rb

```diff
@@ -20,7 +20,18 @@ class GraphqlControllerTest < ActionDispatch::IntegrationTest
     setup do
       account = FactoryBot.create(:account)
       @current_user = FactoryBot.create(:user, account: account)
-      stub_rbac_permissions(Rbac::COMPLIANCE_VIEWER, Rbac::INVENTORY_VIEWER)
+      stub_rbac_permissions(
+        [
+          {
+            permission: Rbac::COMPLIANCE_ADMIN,
+            resource_definitions: Rbac::GLOBAL_RESOURCE_PERMISSION
+          },
+          {
+            permission: Rbac::INVENTORY_VIEWER,
+            resource_definitions: Rbac::GLOBAL_RESOURCE_PERMISSION
+          }
+        ]
+      )
```

Comment:
This seems very repetitive across the app. Could we move the second element into the `stub_rbac_permissions` method instead? At least in cases when it is unset? To have a default GLOBAL value...